src/lib/pull-request.coffee

```diff
@@ -25,6 +25,19 @@ pr = (token, branch, body, cb) ->
   If I do something wrong, [blame a human](https://github.com/cfpb/hubot-capital-framework/issues/new).
 
   ![kitten gif](http://thecatapi.com/api/images/get?format=src&type=gif)
+
+  ## You are not
+
+  After this PR is merged, you should be prepared to update the CF dependencies I've released in the following projects:
+
+  - [ ] [capital-framework gh-pages branch](https://github.com/cfpb/capital-framework/tree/gh-pages) - 
+        The documentation site for Capital Framework, entire repo could be affected.
+  - [ ] [design-manual](https://github.com/cfpb/design-manual/) - 
+        The documentation site for the Design Manual, entire repo could be affected.
+  - [ ] [generator-cf](https://github.com/cfpb/generator-cf/) - 
+        The Yeoman generator for creating new CFPB projects, only the app/templates directory should be affected.
+  - [ ] [cfgov-refresh](https://github.com/cfpb/cfgov-refresh/) - 
+        The CFPB website, cfgov/jinja2/v1, cfgov/unprocessed, and cfgov/wellbeing could all be affected.
```

Comment:
Maybe also add a note that in rare cases (like the icon font to SVG transition), Gulp scripts in all of these projects may be affected.

Reply:
How bout meow?